packages/auth-construct/src/types.ts

```diff
@@ -421,7 +425,10 @@ export type AuthProps = {
      * SES configurations enable the use of customized email sender addresses and names
      * @see https://docs.amplify.aws/react/build-a-backend/auth/moving-to-production/#email
```

Comment:
You will probably want to update this docs section with the parts that I left out when initially implementing senders -- 
this was the full docs message:
     Configure Cognito to send emails from SES or a custom message trigger
     SES configurations enable the use of customized email sender addresses and names
     Custom message triggers enable the use of third-party email providers when sending email notifications to users
     @see https://docs.amplify.aws/react/build-a-backend/auth/moving-to-production/#email
     @see https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-email-sender.html